ENIGMAsystem/SHELL/Widget_Systems/General/tinyfiledialogs/dialogs.cpp

```diff
@@ -364,31 +309,27 @@ string get_open_filenames_ext(string filter, string fname, string dir, string ti
 }
 
 string get_save_filename_ext(string filter, string fname, string dir, string title) {
-  string fname_or_dir;
+  string str_fname_or_dir = "";
   string str_fname = fname;
-  string str_dir;
+  string str_dir = dir;
```

Comment:
It seems that `str_dir` no longer serves a purpose. Did you mean it to be used in some other way than just `dir`?